### PR TITLE
feat(tools): add links to the tools description

### PR DESCRIPTION
**Pull request by xjacka**

### CODE REVIEW

File: src/tools/tools.service.ts [hunk 1]
```diff
@@ -513,7 +513,7 @@ export async function listTools({
             inputSchema: arXivTool.inputSchema,
             isExternal: true,
             userDescription:
-              'Retrieve abstracts of research articles published on ArXiv.org, along with their titles, authors, publication dates, and categories. Ideal for retrieving high-level information about academic papers. The full text of articles is not provided, making it unsuitable for full-text searches or advanced analytics.'
+              'Retrieve abstracts of research articles published on [ArXiv.org](http://arxiv.org), along with their titles, authors, publication dates, and categories. Ideal for retrieving high-level information about academic papers. The full text of articles is not provided, making it unsuitable for full-text searches or advanced analytics.'
```
Comment (pilartomas) [inline src/tools/tools.service.ts:516]: Prefer secure protocol

**Review by kapetr — APPROVED** [review]

**Review by pilartomas — CHANGES_REQUESTED** [review]

File: src/tools/tools.service.ts (hunk 1 above)
Comment (xjacka) [reply src/tools/tools.service.ts:516]: OK, It is copied from the issue.

File: src/tools/tools.service.ts [hunk 2]
```diff
@@ -493,7 +493,7 @@ export async function listTools({
             inputSchema: wikipediaTool.inputSchema,
             isExternal: true,
             userDescription:
-              'Retrieve detailed information from Wikipedia.org on a wide range of topics, including famous individuals, locations, organizations, and historical events. Ideal for obtaining comprehensive overviews or specific details on well-documented subjects. May not be suitable for lesser-known or more recent topics. The information is subject to community edits which can be inaccurate.'
+              'Retrieve detailed information from [Wikipedia.org](https://wikipedia.org) on a wide range of topics, including famous individuals, locations, organizations, and historical events. Ideal for obtaining comprehensive overviews or specific details on well-documented subjects. May not be suitable for lesser-known or more recent topics. The information is subject to community edits which can be inaccurate.'
```
Comment (kapetr) [inline src/tools/tools.service.ts:496]: We want to open it in a new window, this should work `{:target="_blank"}` , but let me try first.

Comment (kapetr) [reply src/tools/tools.service.ts:496]: Nevermind, it doesn't work, we'll add it on the UI side.

**Pull request merged**